provd/proto/keyboard.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package keyboard;
+
+service KeyboardService {
+    rpc SetKeyboard(SetKeyboardRequest) returns (Empty) {}
+    rpc SetInputSource(SetInputSourceRequest) returns (Empty) {}
+}
+
+message Empty {}
+
+message KeyboardSettings {
+    string layout = 1;
+    string varient = 2;
+}
+
+message SetKeyboardRequest {
+    KeyboardSettings settings = 1;
+}
+
+message SetInputSourceRequest {
+  KeyboardSettings settings = 1;
+  string user = 2;
```

Comment:
Is"user" necessary? When its running before the first user is created there is no user yet ... and if its the 2nd user that's created surely we can infer which user this relates to?

Reply:
I see in GIS there seems to be a check for when we are creating a new user, and if we are it sets the settings against that users GSettings. Otherwise it will attempt a system wide setting of the keyboard. 

Looking at the dart code, it looks like we grab the user from environment variables when setting it at a user level, so it should be safe to remove this and move it to the BE :+1: